src/components/core/PivotTable.tsx

```diff
@@ -176,14 +260,19 @@ export const getGridDataSource = (
                     }
                     const { columnDefs, rowData } = executionToAGGridAdapter(
                         execution,
+                        resultSpecWithSorting,
                         {
                             addLoadingRenderer: 'loadingRenderer',
                             columnDefOptions
                         }
                     );
-                    const lastRow = execution.executionResult.paging.total[0];
-                    successCallback(rowData, lastRow);
+                    const { offset, count, total } = execution.executionResult.paging;
+                    // Backend returns incorrectly total: [1, N], when count: [0, N] and offset: [0, N]
```

Comment:
is there any ticket or timeline when it will be fixed?

Reply:
RAIL-1130, přidal jsem to do komentáře